### DISCUSSION TITLE
Wrong title on lottery lesson

### DISCUSSION THREAD

**Discussion by worca333** [Cyfrin/foundry-full-course-cu#1932] [General]
Hello @PatrickAlphaC,

I've been a dedicated learner of your Cyfrin updraft course over the last few weeks. In advance, I'd like to thank you for an amazing content and sweet experience.

Meanwhile, I wanted to note about a small error with a title in your course. As I read, it should be `warp`, not `wrap`. Here's the link.
https://updraft.cyfrin.io/courses/foundry/smart-contract-lottery/vm-roll-warp?lesson_format=video

**Comment by EngrPips** [comment]
Thanks for this heads-up, @worca333. This should be attended to appropriately as soon as possible.
